recipes/simplejsoncpp/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "simplejsoncpp" %}
```

Comment:
We typically call C++ libraries nowadays `libsimplejson` and don't use a `cpp` suffix anymore. It doesn't seem that the upstream code is yet using anywhere `cpp`, so this might be the more favorable option.

Reply:
I changed the name to libsimplejson. It works with the exception of win_64. Is there any way to skip this? It works under linux+mac.